README.md

```diff
@@ -1 +1,48 @@
-# scriptless-percy
+# Scriptless Percy
+
+Sometimes websites are static in nature and to be able capture snapshot of different pages on website using percy without writing a piece of code the sitemaps can be used. Percy by default supports [sitemaps](https://docs.percy.io/docs/cli-snapshot#commands). In case you don't have sitemaps on your websites and still want to use percy without mentioning any url by yourself then this repo can be useful. 
```

Comment:
let's be a little more clear on this:

`use percy without mentioning any url by yourself`

That phrase in the sentence may be confusing to a first time reader.

Reply:
In order to create a sitemap or a configuration file for Percy to use with static sites one needs to specify all the Page URLs for the website. Using this repo one doesn't have to do that hence `use Percy without mentioning any url by yourself`. Let me know if there can be any other better way on how we can convey this ease of use. Also, this tool reads URL's from the in memory Sitemap that is generated and not after file is generated. This approach also allows us to get broken links before every run.